KitchenSink/VideoAudioSetupViewController.swift

```diff
@@ -203,6 +216,19 @@ class VideoAudioSetupViewController: BaseViewController {
             globalVideoSetting.audioBNRMode = .LP
         }
     }
+    
+    @objc func handleVideoStreamGestureEvent(sender:UITapGestureRecognizer) {
+        if sender.view == compositedImageView {
+            compositedImageView.image = checkImage
+            multiStreamImageView.image = uncheckImage
+            webexSDK?.phone.videoStreamMode = .composited
+        }else {
+            compositedImageView.image = uncheckImage
+            multiStreamImageView.image = checkImage
+            globalVideoSetting.audioBNRMode = .LP
```

Comment:
Is the BNR mode change required?

Reply:
Thanks @akshayagarwaldev, I will remove it.